src/main/java/uk/gov/hmcts/reform/civil/notification/handlers/changeofrepresentation/common/NotificationHelper.java

```diff
@@ -0,0 +1,212 @@
+package uk.gov.hmcts.reform.civil.notification.handlers.changeofrepresentation.common;
+
+import uk.gov.hmcts.reform.ccd.model.Organisation;
+import uk.gov.hmcts.reform.ccd.model.OrganisationPolicy;
+import uk.gov.hmcts.reform.civil.enums.CaseRole;
+import uk.gov.hmcts.reform.civil.model.CaseData;
+import uk.gov.hmcts.reform.civil.model.RecipientData;
+
+import java.util.Optional;
+
+import static uk.gov.hmcts.reform.civil.enums.MultiPartyScenario.TWO_V_ONE;
+import static uk.gov.hmcts.reform.civil.enums.MultiPartyScenario.getMultiPartyScenario;
+
+public class NotificationHelper {
+
+    private NotificationHelper() {
+        //NO-OP
+    }
+
+    public static final String LIP = "LiP";
+
+    public static String getCaseName(CaseData caseData) {
+        String applicants = caseData.getApplicant1().getPartyName();
+        if (caseData.getApplicant2() != null) {
+            applicants += String.format(", %s", caseData.getApplicant2().getPartyName());
+        }
+        String defendants = caseData.getRespondent1().getPartyName();
+        if (caseData.getRespondent2() != null) {
+            defendants += String.format(", %s", caseData.getRespondent2().getPartyName());
+        }
+
+        return String.format("%s v %s", applicants, defendants);
+    }
+
+    public static String getOtherSolicitor2Name(CaseData caseData) {
+        RecipientData otherSolicitor2 = getOtherSolicitor2(caseData);
+        if (otherSolicitor2 != null) {
+            return otherSolicitor2.getOrgId();
+        }
+        return null;
+    }
+
+    /**
+     * Gets the other solicitor 2 to compliment other solicitor 1 if its a 1v1. if its a 2v1 it'll get the applicant 1
+     * solicitor as oppose to applicant2 as theyre the same solicitor.
+     * @param caseData data that contains which party has changed
+     * @return Recipient data model that contains a String email and a String Organisation id, else returns null if
+     *     the part is a LiP.
+     */
+    protected static RecipientData getOtherSolicitor2(CaseData caseData) {
```

Comment:
Looks like there is a lot of duplication in the methods here between OtherSolcitor1 and OtherSolicitor2. We should abstract these common methods out and use inheritance where possible here